include/invert_quda.h

```diff
@@ -798,12 +818,17 @@ namespace quda {
     /**
        @brief Compute the alpha coefficients
     */
-    void compute_alpha(Complex *x, Complex *W, Complex *phi);
+    void compute_alpha();
 
     /**
        @brief Compute the beta coefficients
     */
-    void compute_beta(Complex *beta, Complex *W, Complex *C);
+    void compute_beta();
+
+    /**
+       @ brief Check if it's time for a reliable update
+    */
+    int reliable(double &rNorm,  double &maxrr, int &rUpdate, const double &r2, const double &delta);
```

Comment:
clang-format suggested the following change:


```
-    int reliable(double &rNorm,  double &maxrr, int &rUpdate, const double &r2, const double &delta);
+    int reliable(double& rNorm, double& maxrr, int& rUpdate, const double& r2, const double& delta);
```

<!-- identifier: clang-format -->
<!-- File: include/invert_quda.h -->